.github/workflows/linux.yml

```diff
@@ -163,6 +163,8 @@ jobs:
                 -DCMAKE_CXX_COMPILER=${compilers[1]}       \
                 -G"Ninja"                                  \
                 -DYACLIB_LINK_OPTIONS="$link_option"       \
+                -DYACLIB_FAULT=OFF                         \
+                -DYACLIB_LOG="INFO"                        \
```

Comment:
should be ERROR (in test we check some stupid code, that trigger info log)

Reply:
but then tests for info logs won't work

Reply:
Maybe we make separate target for test logging?

Reply:
well actually they won't be run in that case